test/e2e/framework/kubernetes/get-external-crd.go

```diff
@@ -0,0 +1,62 @@
+package kubernetes
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+func downloadExternalCRDs(chartPath string) error {
+	crdUrls := []string{
+		"https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml",
+	}
+
+	for _, crdUrl := range crdUrls {
+		crd, err := fetchYAML(crdUrl)
+		if err == nil && crd != nil {
```

Comment:
error handling in go is normally `if err != nil {}` then happy path. Please remove `if err == nil`